script.js

```diff
@@ -0,0 +1,14 @@
+const hamburger = document.querySelector(".hamburger");
+const navMenu = document.querySelector(".nav-menu");
+
+hamburger.addEventListener("click", () => {
+  hamburger.classList.toggle("active");
+  navMenu.classList.toggle("active");
+});
+
+document.querySelectorAll(".nav-link").forEach((n) =>
+  n.addEventListener("click", () => {
+    hamburger.classList.remove("active");
+    navMenu.classList.remove("active");
+  })
+);
```

Comment:
- [ ] Please make sure when the user clicks on any of the options in the mobile menu it takes them to the appropriate part of the webpage 🙏 as stated in the project requirement.

![20220719-163158_capture](https://user-images.githubusercontent.com/74814780/179791453-cf12a357-06b8-4e63-ab81-e2ce80e137d7.gif)